client/components/data/query-user-purchases/index.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
```

Comment:
You're right, but the thing is, `render` returns `null` in all Query Components and no JSX is ever used usually.
So I'm not sure this is really needed here.